### PR TITLE
[IOTDB-4080] Change the port name of show cluster, show regions, show confignodes, show datandoes; make running result of `Remove-datanode.sh` clear

### PR DESCRIPTION
**Pull request by Beyyes**
## Description

Change `port` to `InternalPort` or `RpcPort` of the show cluster, show regions, show confignodes, show datandoes.

image

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java [hunk 1]
```diff
@@ -68,6 +68,8 @@ public class ColumnHeaderConstant {
   public static final String COLUMN_STATUS = "Status";
   public static final String COLUMN_HOST = "Host";
   public static final String COLUMN_PORT = "Port";
+  public static final String COLUMN_INTERNAL_PORT = "InternalPort";
+  public static final String COLUMN_RPC_PORT = "RpcPort";
```
Comment (CRZbulabula) [inline server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java:72]: Is the static field COLUMN_PORT can be deleted now?

File: server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java [hunk 2]
```diff
@@ -200,7 +202,7 @@ public class ColumnHeaderConstant {
           new ColumnHeader(COLUMN_SERIES_SLOTS, TSDataType.INT64),
           new ColumnHeader(COLUMN_TIME_SLOTS, TSDataType.INT64),
           new ColumnHeader(COLUMN_DATANODE_ID, TSDataType.INT32),
-          new ColumnHeader(COLUMN_HOST, TSDataType.TEXT),
+          new ColumnHeader(COLUMN_RPC_PORT, TSDataType.TEXT),
           new ColumnHeader(COLUMN_PORT, TSDataType.INT32),
```
Comment (CRZbulabula) [inline server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java:206]: COLUMN_RPC_PORT and COLUMN_PORT? Is there exist any failure?

**Review by CRZbulabula — CHANGES_REQUESTED** [review]
PTAL~

**Review by qiaojialin — APPROVED** [review]

File: confignode/src/main/java/org/apache/iotdb/confignode/procedure/env/DataNodeRemoveHandler.java [hunk 3]
```diff
@@ -507,7 +507,7 @@ private TSStatus checkRegionReplication(RemoveDataNodePlan removeDataNodePlan) {
     if (allDataNodeSize - removedDataNodeSize < NodeInfo.getMinimumDataNode()) {
       status.setCode(TSStatusCode.LACK_REPLICATION.getStatusCode());
       status.setMessage(
-          "lack replication, allow most removed Data Node size : "
+          "due to the limit of replication factor, max allowed removed Data Node size is: "
```
Comment (qiaojialin) [inline confignode/src/main/java/org/apache/iotdb/confignode/procedure/env/DataNodeRemoveHandler.java:510]: Print the replication factor

File: server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java (hunk 2 above)
Comment (Beyyes) [reply server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java:206]: i will remove is and add test screenshot

File: server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java (hunk 1 above)
Comment (Beyyes) [reply server/src/main/java/org/apache/iotdb/db/mpp/common/header/ColumnHeaderConstant.java:71]: delete

**Comment by snake-zhiqiang** [comment]
Msg: 401: Error occurred while parsing SQL to physical plan: line 1:5 no viable alternative at input 'show cluster'

**Pull request merged**